app-crypt/snow/snow-20130616.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Hides information in text files using trailing whitespace"
+DESCRIPTION="SNOW (Steganographic Nature Of Whitespace) is a steganography \
+tool which conceals messages in ASCII text by appending spaces to the end of \
+lines. Has an option to encrypt the message first to further secure it."
+HOMEPAGE="http://www.darkside.com.au/snow/"
+COPYRIGHT="Matthew Kwan"
```

Comment:
We like to include the copyright years, if available. That would be something in the following format:
```
COPYRIGHT="YYYY-20YY Matthew Kwan"
```